backup_and_restore/control_plane_backup_and_restore/disaster_recovery/about-disaster-recovery.adoc

```diff
@@ -42,3 +44,9 @@ expired. For example, if you shut down your cluster before the first certificate
 rotation, which occurs 24 hours after installation, your certificates will not
 be rotated and will expire. You can follow this procedure to recover from
 expired control plane certificates.
+
```

Comment:
Should testing be it's own linked page?

Reply:
I believe it makes sense placing it here. Linking it to another page like the restoration procedures above would be more confusing IMO.

Reply:
Fair enough